### ISSUE TITLE
[Core] findcog

### ISSUE THREAD

**Issue by Stonedestroyer** [Cog-Creators/Red-DiscordBot#4032]
# Command bugs

#### Command name

findcog

#### What cog is this command from?

Downloader

#### What were you expecting to happen?

The command to show

Use findcog on a custom added cog that has no info.json.

```py
Exception in command 'findcog'
Traceback (most recent call last):
  File "C:\Data\Red\CanaryVenv-3.8.2\lib\site-packages\discord\ext\commands\core.py", line 83, in wrapped
    ret = await coro(*args, **kwargs)
  File "C:\Data\Red\CanaryVenv-3.8.2\lib\site-packages\redbot\cogs\downloader\downloader.py", line 1392, in findcog
    made_by = humanize_list(cog_installable.author) or _("Missing from info.json")
  File "C:\Data\Red\CanaryVenv-3.8.2\lib\site-packages\redbot\core\utils\chat_formatting.py", line 390, in humanize_list
    raise IndexError("Cannot humanize empty sequence") from None
IndexError: Cannot humanize empty sequence

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Data\Red\CanaryVenv-3.8.2\lib\site-packages\discord\ext\commands\bot.py", line 892, in invoke
    await ctx.command.invoke(ctx)
  File "C:\Data\Red\CanaryVenv-3.8.2\lib\site-packages\discord\ext\commands\core.py", line 797, in invoke
    await injected(*ctx.args, **ctx.kwargs)
  File "C:\Data\Red\CanaryVenv-3.8.2\lib\site-packages\discord\ext\commands\core.py", line 92, in wrapped
    raise CommandInvokeError(exc) from exc
discord.ext.commands.errors.CommandInvokeError: Command raised an exception: IndexError: Cannot humanize empty sequence
```

Add a custom cog
use findcog on it.

Haven't tried non-embeds.

**Comment by Vexed01** [comment]
Oh no this must be because of my changes to findcog...

**Comment by Jackenmen** [comment]
If nobody fixes this until 3.4 release, this is gonna get auto-fixed by the changes to `humanize_list()`
This is gonna be a regression in 3.3.10 so anyone can feel free to make a fix for it, but I'm leaving this be.
